spec/openapi.yaml

```diff
@@ -3950,6 +3997,299 @@ components:
+                    id:
+                      description: Address identifier
+                      type: string
+                      format: uuid
+                    name:
+                      properties:
+                        first:
+                          description: First name
+                          type: string
+                        last:
+                          description: Last name
+                          type: string
+                    company:
+                      description: Company name
+                      type: string
+                      nullable: true
+                    street:
+                      properties:
+                        name:
+                          description: Street name
+                          type: string
+                        number:
+                          description: Street number
+                          type: string
+                        suffix:
+                          description: Street number suffix
+                          type: string
+                          nullable: true
+                    zipCode:
+                      description: Zip code
+                      type: string
+                    city:
+                      description: City
+                      type: string
+                    countryCode:
+                      description: ISO 3166-1 alpha-2 country code
+                      type: string
+                      pattern: '^[A-Z]{2}$'
+                    phone:
+                      description: Phone number
+                      type: string
+                      nullable: true
+                    gender:
+                      description: Gender
+                      type: string
+                      nullable: true
+                      enum:
+                        - F
+                        - M
+                invoiceAddress:
+                  description: Invoice address of the customer
+                  type: object
+                  properties:
+                    id:
+                      description: Address identifier
+                      type: string
+                      format: uuid
+                    firstName:
+                      description: First name
+                      type: string
+                    lastName:
+                      description: Last name
+                      type: string
+                    company:
+                      description: Company name
+                      type: string
+                      nullable: true
+                    streetName:
+                      description: Street name
+                      type: string
+                    streetNumber:
+                      description: Street number
+                      type: string
+                    streetSuffix:
+                      description: Street number suffix
+                      type: string
+                      nullable: true
```

Comment:
Both `street` and `name` of invoice address should also be split up in separate sub keys, like was done at the delivery address.